eiffel-vocabulary/EiffelIssueDefinedEvent.md

```diff
@@ -0,0 +1,251 @@
+<!---
+   Copyright 2018 Jaden Young
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+--->
+
+# EiffelIssueDefinedEvent (ID)
+The EiffelIssueDefinedEvent declares that an issue has been created in some
+external issue management software. This event provides provenance for an issue,
+and acts as a "handle" for future issue related events similarly to
+[EiffelActivityTriggeredEvent](./EiffelActivityTriggeredEvent.md) and 
+[EiffelTestCaseTriggeredEvent](./EiffelTestCaseTriggeredEvent.md). It provides
+a minimal amount of information necessary for this purpose and allows further
+data to be attached through later events, for example the 
+[EiffelIssueAssignedEvent](./EiffelIssueAssignedEvent).
+
+## Data Members
```

Comment:
Should ID include an initial status?

Reply:
We can definitely add it. It's a trade off of redundant fields in ID and ISM vs clarity of initial state.